test/app/components/market/submit/parameter-input.component.spec.ts

```diff
@@ -130,6 +130,71 @@ describe("ParameterInputComponent", () => {
             fixture.detectChanges();
             expect(inputEl.nativeElement.value).toBe(updatedInput);
         });
+
+        it("should validate minimum length constraint", () => {
```

Comment:
You should also check for the error message

Reply:
done